modules/gdscript/gdscript_cache.cpp

```diff
@@ -151,22 +151,31 @@ void GDScriptCache::move_script(const String &p_from, const String &p_to) {
 		return;
 	}
 
-	if (singleton->parser_map.has(p_from) && !p_from.is_empty()) {
+	if (singleton->parser_map.has(p_from)) {
 		singleton->parser_map[p_to] = singleton->parser_map[p_from];
 	}
 	singleton->parser_map.erase(p_from);
 
-	if (singleton->parser_inverse_dependencies.has(p_from) && !p_from.is_empty()) {
+	if (singleton->dependencies.has(p_from)) {
+		singleton->dependencies[p_to] = singleton->dependencies[p_from];
+	}
+	singleton->dependencies.erase(p_from);
```

Comment:
This isn't strictly relevant to the issue, but it makes sense that it is remapped along with the other hashmaps here.